src/popover/arrow.scss

```diff
@@ -71,10 +71,10 @@
   }
 
   &-inner {
-    inset-block-start: 2px;
+    inset-block-start: awsui.$border-width-field;
```

Comment:
classic `border-width-field` is 1px as far as I can see, so I think both should be `calc(awsui.$border-width-field + 1px);`?

Reply:
Good catch. They can both share the same inset value now that it is tokenized.